src/lib/flight_tasks/tasks/Utility/Sticks.hpp

```diff
@@ -53,7 +53,7 @@ class Sticks : public ModuleParams
 	Sticks(ModuleParams *parent);
 	~Sticks() = default;
 
-	void evaluateSticks(hrt_abstime now); ///< checks and sets stick inputs
+	bool evaluateSticks(hrt_abstime now); ///< checks and sets stick inputs
```

Comment:
We could also rename it `checkAndSetStickInputs` and get rid of the comment.... :D

Reply:
I did that also to the other function, see last commit. Also rebased on master since CI is succeeding again. Sadly I lost the review and am not allowed anymore to override and merge 🤷